### DISCUSSION TITLE
Updated file not showing on live site

### DISCUSSION THREAD

**Discussion by tengrrl** [community/community#23226] [Other Feature Feedback, Questions, & Ideas]
<p>I have a static Pages site that has been working for a year and a half. My process is editing the files locally and then committing and pushing the files to the repository, with Github Desktop on Windows 10. The changes are normally live in 5 to 10 minutes. I typically flush the cache for the page to see the changes in my browser (Chrome).</p>
<p>I made a change several hours ago to a file locally, committed and pushed the file. Even though hours have passed, the changes are not live. The changed files DO show in the repo though. So the file has been added properly, but it’s not showing as live. The site is still there and working just fine, but this updated file will not load.</p>
<p>I have tried changing various things and pushing a new version. I have tried in Chrome, in a Chrome Incognito window, and with all the cache deleted. I also tried in Firefox. To make sure it wasn’t something on my machine, I tried on an iPad and I tried on my phone (so a different network connection). None of these options show the revised file, even though that’s what’s showing in the repo.</p>
<p>Has something changed in the last 2 or 3 months for GitHub Pages? I had a medical issue and I this is the first time recently that I have worked on the site. Anyone have any tips or suggestions? I am at a loss.</p>
<p>The problem page: <a href="https://tracigardner.github.io/TechComm/genres/how-to-email.html" rel="noopener nofollow ugc">https://tracigardner.github.io/TechComm/genres/how-to-email.html</a><br>
The repo: <a href="https://github.com/tracigardner/tracigardner.github.io" rel="noopener nofollow ugc">https://github.com/tracigardner/tracigardner.github.io</a></p>

**Comment by tengrrl** [comment]
<p>Still trying to figure this problem out. I have checked the commits and found that the page builds aren’t happening. Nothing has built since Nov 29 (the last time I added files).</p>
<p>Unfortunately, I am lost on why it’s not building. I have a cron job that seems to be working. It emails me messages that say the process is queued, but nothing is building. There has to be something other than HTML changes going on, but I can’t figure out what it is that has changed.</p>
<p>Any suggestions?</p>

**Comment by tengrrl** [comment, marked as answer]
<p>I’ve given up on figuring this out. I renamed the repo, created it again with the original name, and uploaded all of the files. New version is working. No idea why it broke.</p>

**Comment by joshcarst** [comment]
<p>I’m also having this issue currently. In the deployments activity log it’s not showing any new deployments by github-pages regardless of new changes I commit to <a href="https://github.com/joshcarst/joshcarst.github.io" rel="noopener nofollow ugc">my repo</a>, with the last successful one occurring on Sept 11th of last year. I suppose I’ll also try doing what you did but it’d be nice if someone at GitHub could be notified.</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/joshcarst">@joshcarst</a>, welcome to the GitHub Support Community! Looking at the logs of builds for your site it’s showing the latest commit has successfully built. It also looks like all your recent commits have successful builds too.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/8/a848acd1fcb5396336f35852c0cfc0b0166d9ebb.png" data-download-href="/uploads/short-url/o0HV54usmiy7q9FeetkdfqUVMSL.png?dl=1" title="Screenshot 2021-01-06 at 12.04.21"><img src="https://user-images.githubusercontent.com/2439407/181075657-54c64c42-dc05-41c5-8788-65dd9ac2b6ef.png" alt="Screenshot 2021-01-06 at 12.04.21" data-base62-sha1="o0HV54usmiy7q9FeetkdfqUVMSL" width="690" height="187" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/8/a848acd1fcb5396336f35852c0cfc0b0166d9ebb_2_690x187.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/8/a848acd1fcb5396336f35852c0cfc0b0166d9ebb_2_1035x280.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/8/a848acd1fcb5396336f35852c0cfc0b0166d9ebb_2_1380x374.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/8/a848acd1fcb5396336f35852c0cfc0b0166d9ebb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-01-06 at 12.04.21</span><span class="informations">1702×462 71.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Are you having trouble with a specific page or specific change not showing up? If so, can you post the specific link or a screenshot of the issue?</p>

**Comment by joshcarst** [comment]
<p>I should have clarified, I did as <a class="mention" href="/tengrrl">@tengrrl</a> mentioned and created a new repo and re-uploaded all my site files to it which seems to have solved the issue. I deleted the original repo which was giving me trouble, although I probably should have kept it around for diagnosis. I greatly appreciate you taking a look at it though.</p>

**Comment by toygma** [comment]
I tried many times but it doesn't work strange error
